circle.yml

```diff
@@ -123,6 +124,17 @@ commands:
               --npm npm-package-url.json \
               --binary binary-url.json
 
+  verify-mocha-results:
+    description: Double-check that Mocha tests ran as expected.
+    parameters:
+      expectedResultCount:
+        description: The number of result files to expect, ie, the number of Mocha test suites that ran.
+        type: integer
+        ## by default, assert that at least 1 test ran
+        default: 0
```

Comment:
is this supposed to be 0 or 1

Reply:
0, if this is falsy, it means "assert at least 1 test ran", otherwise, it means "assert exactly N tests ran"

Reply:
ah i see it just has to be falsey